modules/ting_search_autocomplete/includes/ting_search_autocomplete.admin.inc

```diff
@@ -141,37 +28,15 @@ function ting_search_autocomplete_admin() {
     '#title' => t('Autocomplete help', array(), array('context' => 'ting_search_autocomplete')),
     '#tree' => TRUE,
     '#collapsible' => TRUE,
-    '#collapsed' => TRUE,
+    '#collapsed' => FALSE,
```

Comment:
The complex structure of settings and help is unnecessary with so few settings remaining. Please reduce them to unnested form elements with standard descriptions.

Reply:
There's only two elements:

![screen shot 2017-11-16 at 11 23 02](https://user-images.githubusercontent.com/514013/32886333-aa6a31d6-cac0-11e7-83b9-aa9ea1aac554.png)

I think it's good in this case.

Reply:
Still applies.